sdk/mongocluster/arm-mongocluster/src/rest/documentDBClient.ts

```diff
@@ -57,9 +57,8 @@ export default function createClient(
       // Append one if there is no apiVersion and we have one at client options
       const url = new URL(req.url);
       if (!url.searchParams.get("api-version") && apiVersion) {
-        req.url = `${req.url}${
-          Array.from(url.searchParams.keys()).length > 0 ? "&" : "?"
-        }api-version=${apiVersion}`;
+        req.url = `${req.url}${Array.from(url.searchParams.keys()).length > 0 ? "&" : "?"
+          }api-version=${apiVersion}`;
```

Comment:
what is causing the changes here? Looks like a formating issue.

Reply:
yes